examples/examples-gpu/nyc-taxi/rf-rapids.ipynb

```diff
@@ -4,47 +4,98 @@
    "cell_type": "markdown",
```

Comment:
<p>is there a reason to get the <code>features</code>  like this way instead of using the one defined in the <code>prep_df()</code>  function? The list could be moved to notebook scope then would be accessible here</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/saturncloud/examples/pull/58/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='25'/>

Reply:
<p>I think it's a cleaner contract to say "<code>prep_df()</code>  produces a data frame with only the features you want to use + the target". The current code on <code>main</code>  is referencing global variables like <code>features</code>  inside the body of <code>prep_df()</code>  , which I think makes it harder to follow and reason about.</p><p> </p><p>Doing it the way I've proposed in this PR makes each step self-contained, which I think is important for tutorials like this that you do one step at a time.</p>